harness/determined/common/experimental/determined.py

```diff
@@ -155,32 +155,36 @@ def create_model(
 
         return model.Model.from_json(r.json().get("model"), self._session)
 
-    def get_model(self, model_id: int) -> model.Model:
+    def get_model(self, name: str) -> model.Model:
         """
         Get the :class:`~determined.experimental.Model` from the model registry
-        with the provided id. If no model with that id is found in the registry,
+        with the provided name. If no model with that id is found in the registry,
         an exception is raised.
         """
-        r = self._session.get("/api/v1/models/{}".format(model_id))
+        r = self._session.get("/api/v1/models/{}".format(name))
         return model.Model.from_json(r.json().get("model"), self._session)
 
-    def get_model_by_name(self, name: str) -> model.Model:
+    def get_model_by_id(self, model_id: int) -> model.Model:
         """
         Get the :class:`~determined.experimental.Model` from the model registry
-        with the provided name. If no model with that name is found in the registry,
+        with the provided id. If no model with that id is found in the registry,
         an exception is raised.
         """
-        r = self._session.get("/api/v1/models?name={}".format(name))
+        r = self._session.get("/api/v1/models?id={}".format(model_id))
         models = r.json().get("models")
         assert len(models) > 0
         return model.Model.from_json(models[0], self._session)
 
+    def get_model_by_name(self, name: str) -> model.Model:
+        return self.get_model(name)
+
```

Comment:
I'd prefer we not introduce alias functions unnecessarily.  I think `get_model(name)` is intuitive enough we can skip this.

Reply:
I've removed `get_model_by_name`